pytransform3d/transform_manager/_transform_graph_base.py

```diff
@@ -37,6 +39,7 @@ def __init__(self, strict_check=True, check=True):
         # distance matrix (distance is the number of transformations)
         self.dist = np.empty(0)
         self.predecessors = np.empty(0, dtype=np.int32)
+        self._transforms = {}
```

Comment:
As the type of this dictionary is different between the normal and temporal TransformManager, I wouldn't include it in the base class.